modules/ROOT/pages/vantagecloud-lake-demos-visual-studio-code.adoc

```diff
@@ -0,0 +1,111 @@
+
+image::vscode.png[vscode.png,align="center", width=80%]
+
+== Prerequisites
+Before you begin, ensure you have the following prerequisites in place:
+
+* https://www.docker.com/products/docker-desktop/[Docker Desktop] installed
+* https://git-scm.com/book/en/v2/Getting-Started-Installing-Git[Git] installed
+** Required to download git repo from https://github.com/Teradata/lake-demos.git
+* https://code.visualstudio.com/download[Visual Studio Code] installed
+* A Teradata VantageCloud Lake account with organization URL and login details from the Teradata welcome letter
+
+
+== Clone VantageCloud Lake Demo repository 
+Begin by cloning the GitHub repository and navigating to the project directory:
+----
+git clone https://github.com/Teradata/lake-demos.git
+cd lake-demos
+----
+== Mount Docker files 
+To launch Teradata VantageCloud Lake demos, we need the https://hub.docker.com/r/teradata/jupyterlab-extensions[Jupyter extension for Docker]. This extension bundles Teradata SQL kernel, Teradata Python, R libraries, drivers and Teradata extensions for seamless interaction with Teradata databases. 
+----
+docker pull teradata/jupyterlab-extensions
+----
+Next, start a container and bind it to the existing lake-demos directory. Replace */path/to/your/directory* with your actual directory:
+----
+docker run -e "accept_license=Y" \
+  -p 127.0.0.1:8888:8888 \
+  -v /path/to/your/directory:/home/jovyan/JupyterLabRoot teradata/jupyterlab-extensions
+----
+Take note of the resulting URL and token; you'll need them to establish the connection from Visual Studio Code.
+
+image::terminal.png[terminal.png,align="center"]
+
+== Visual Studio Code Configuration
+Open `lake-demos` project directory in Visual Studio Code. The repository contains the following project tree: 
+
+LAKE_DEMOS
+
+* https://github.com/Teradata/lake-demos/tree/main/UseCases[UseCases]
+** https://github.com/Teradata/lake-demos/blob/main/0_Demo_Environment_Setup.ipynb[0_Demo_Environment_Setup.ipynb]
+** https://github.com/Teradata/lake-demos/blob/main/1_Load_Base_Demo_Data.ipynb[1_Load_Base_Demo_Data.ipynb]
+** https://github.com/Teradata/lake-demos/blob/main/Data_Engineering_Exploration.ipynb[Data_Engineering_Exploration.ipynb]
+** https://github.com/Teradata/lake-demos/blob/main/Data_Science_OAF.ipynb[Data_Science_OAF.ipynb]
+** https://github.com/Teradata/lake-demos/blob/main/Demo_Admin.ipynb[Demo_Admin.ipynb]
+* https://github.com/Teradata/lake-demos/blob/main/vars.json[vars.json file]
+
+Navigate to the *vars.json* file and include the following environment variables:
+
+----
+   "environment":{"host":"XXX.XXX.XXX.XXX", 
+                   "UES_URI":"", 
+                   "bucket":"", 
+                   "access_key":"", 
+                   "secret":""},    
```

Comment:
Just making a note, this will change once Daniel shares the coordinates of the newly created public bucket and Kevin validates the notebooks against the new bucket.